MPhys/Rich_Geometry/One emitter for One photon .ipynb

```diff
@@ -81,7 +81,7 @@
     "photon_detected.reset_index(drop=True, inplace= True)\n",
     "\n",
     "#Can be removed later. Outputs characteristics of saved photons.\n",
-    "photon_detected.to_csv('./photons_detected_spectral.csv' )\n",
+    "photon_detected.to_csv('./test_photons_detected_spectral.csv' )\n",
```

Comment:
This should probably be written to a csv directory now as done in the Single_Emitter directory - check through for other similar examples.